includes/polling/os/ceraos.inc.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+$hardware = snmp_walk($device, 'MWRM-UNIT-MIB::genEquipInventoryCardName', '-Osqnv');
```

Comment:
Unfortunately with snmp_get it does not work. In fact when I poll the devices I have access to, the value is in genEquipInventoryCardName.127 (for example). Not sure if that 127 is always the same and hence why I used snmp_walk, but i can also do snmp_get with the .127 at the end if you prefer so.